spl/token/instructions.py

```diff
@@ -337,6 +350,14 @@ def decode_initialize_multisig(instruction: TransactionInstruction) -> Initializ
     )
 
 
+def decode_create_associated_token_account(instruction: TransactionInstruction) -> AssociatedTokenAccountParams:
```

Comment:
I looked at solana's JS code and realized this is actually not necessary, since this instruction takes no data.